src/libraries/Common/src/System/IO/ChunkedMemoryStream.cs

```diff
@@ -35,6 +35,13 @@ public byte[] ToArray()
 
         public override void Write(byte[] buffer, int offset, int count)
         {
+            Write(new ReadOnlySpan<byte>(buffer, offset, count));
```

Comment:
This file doesn't have anything to do with the PR title of "DeflateStream.CopyToStream", does it?

Reply:
No, I need to change the PR title. I've put all the "trivial" stream changes in this PR.